templates/core/features/index.html

```diff
@@ -0,0 +1,223 @@
+{% extends "core/base_core.html" %}
+
+{% block title %}Features | Ubuntu Core{% endblock %}
+{% block meta_description %}Case studies and customer stories of products launched with Ubuntu Core, a secure, fully containerised Ubuntu for embedded devices that is optimised for reliable updates.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/10S6FGVRA1R1qi9v859CnFrl8rS25Za2SjEmSpPVQkzY/edit#heading=h.2rlj58ccsw9y{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row">
+    <h1>Ubuntu Core features</h1>
+    <p class="p-heading--4">What’s under the hood</p>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h2>Snaps</h2>
+      <p>Ubuntu Core is built from snaps, a secure, confined, dependency-free, cross-platform Linux packaging format. Snaps are entirely self-contained, even to the point of encapsulating their own file system. This means they include everything they need to run in any environment. They're used by Ubuntu Core to both compose the image that's run on a device, and to deliver consistent and reliable software updates, even to low-powered, inaccessible and remotely administered embedded and IoT systems.</p>
+      <p>
+        <a href="/core/docs/snaps-in-ubuntu-core">Learn more about snaps</a>
```

Comment:
needs the >

Reply:
Great thanks, yeah I'll take note of where we will need to add them